dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -162,10 +175,30 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 			}
 		}
 		params = sortByKey(params);
-
+		boolean byInode = params.containsKey("byInode");

```

Comment:
this could be final

Reply:
No it cannot. That variable can be changed in line 187.